main/acle.md

```diff
@@ -2699,11 +2704,41 @@ following:
   in one of the translation units.
   * Implicitly, without this attribute,
   * or explicitly providing the `default` in the attribute.
-* All instances of the versions shall share the same function
-  signature and calling convention.
+* The default version signature is the signature for calling
+  the multiversioned functions.
+* Non-default versions shall share the same calling convention
+  of the default version and have a type that is convertable to the type
+  of the default version.
+
 * All the function versions must be declared at the translation
   unit in which the definition of the default version resides.
 
+For example, in the case below:
+```C
+__attribute__((target_version("default"))) void fn(void);
+__attribute__((target_version("foo"))) void fn(void); // f_foo below
+````
+
+This case is valid as it is possible to use the
+non-default function as the value for a pointer with the type of
+the default function, as shown below.
+
+```C
+void f_foo(void);
+void (*f)(void) = f_foo;
+```
+
+Additionally, the below example is valid and 2 is passed as the
```

Comment:
I think we can merge these two examples into one if you still want to keep the first? There is no versioned `f_foo` in your first example.
```C++
__attribute__((target_version("default"))) void f(int c = 1);
__attribute__((target_version("sve"))) void f(int);

void (*ptr)(int) = f;

void caller() {
  f(); // passes 1
  ptr(2); // passes 2
}
```

Reply:
I think I'll remove the first one. Thank you!